src/platform-implementation-js/dom-driver/gmail/views/gmail-app-sidebar-view/primary/index.js

```diff
@@ -166,7 +167,7 @@ class GmailAppSidebarPrimary {
       document.createElement('div'));
     threadSidebarContainerEl.classList.add(
       'thread_app_sidebar',
-      idMap('app_sidebar_container'),
+      ...[legacyIdMap, idMap].map((f) => f('app_sidebar_container')),
```

Comment:
We never look for an element with the classname legacyIdMap('app_sidebar_container') produced by another SDK (or the current one), so we don't need legacyIdMap for 'app_sidebar_container'.

Reply:
[5624339](https://github.com/InboxSDK/InboxSDK/pull/895/commits/5624339b7568f255ab9a0b14402cfd3f25e8722c)